maze_transformer/evaluation/plot_maze.py

```diff
@@ -116,9 +122,24 @@ def add_multiple_paths(self, path_list: list):
                 self.add_predicted_path(path=path)
         return self
 
-    def show(self) -> None:
+    def add_node_values(self, node_values: NDArray) -> MazePlot:
```

Comment:
this is very minor, but you shouldn't worry about adding `NDArray` or `ATensor` annotations -- use jaxtyping instead. Fox example, this could be
`jaxtyping.Float[np.ndarray, "grid_n grid_n"`

Reply:
as discussed, leaving it here as is, implementing jaxtyping from now on.